DEVELOPERS.md

```diff
@@ -264,6 +264,12 @@ And developers should feel free to use them wherever this aids clarity vs a docs
 Dataclasses have also retained their annotations to avoid initialising all fields with None.
 
 
+## Documentation
+
+The documentation in this repository forms part of the main [OpenSAFELY documentation](https://github.com/opensafely/documentation).
+
+It can also be built as a standalone documentation site with MkDocs to preview content changes.
+
```

Comment:
Mention how to do that?  i.e.
```just docs-serve```

Reply:
We should probably also mention the things that will look different - formatting, glossary includes (should also note how to do those).  Maybe a to-do for later though

Reply:
Yep, good point; I've opened an issue for the differences in appearance and linked to it.